apps/web/components/v2/eventtype/AvailabilityTab.tsx

```diff
@@ -91,15 +92,16 @@ const AvailabilitySelect = ({
   );
 };
 
-const format = (date: Date) =>
-  Intl.DateTimeFormat(undefined, { hour: "numeric", minute: "numeric" }).format(
+const format = (date: Date, hour12: boolean) =>
+  Intl.DateTimeFormat(undefined, { hour: "numeric", minute: "numeric", hour12 }).format(
     new Date(dayjs.utc(date).format("YYYY-MM-DDTHH:mm:ss"))
   );
 
 export const AvailabilityTab = () => {
   const { t, i18n } = useLocale();
   const { watch } = useFormContext<FormValues>();
-
+  const me = useMeQuery();
+  const { timeFormat } = me.data || { timeFormat: null };
```

Comment:
Perhaps it's just me, but I don't really like the formatting this way 🤔  
I'd rather see something like `const timeFormat = me?.data?.timeFormat`, which also makes it more obvious to me that this data could be undefined. What do you think?

Reply:
done